src/asammdf/blocks/mdf_v4.py

```diff
@@ -6576,8 +6568,6 @@ def attach(
 
         file_path = Path(file_name)
 
-        mime: str | None
-
         mime, _ = guess_type(Path(file_path))
```

Comment:
Hi @VorobiovM, I had this error as well with mypy. I fixed it differently though. It's unclear to me why mime is a function argument although it gets overridden line 6571.

@danielhrisca did you keep the function argument mime for backward compatibility and will remove it in some future version?